docs/sources/flow/reference/components/loki.process.md

```diff
@@ -330,6 +330,30 @@ stage.labels {
 }
 ```
 
+### stage.non_indexed_labels block
+
+The `stage.non_indexed_labels` inner block configures a labels processing stage that can read
+data from the extracted values map and add them to log entries as non-indexed labels.
+
+The following arguments are supported:
+
+| Name     | Type          | Description                                                                 | Default | Required |
+| -------- | ------------- |-----------------------------------------------------------------------------| ------- | -------- |
+| `values` | `map(string)` | Specifies the list of labels to add from extracted values map to log entry. | `{}`    | no       |
+
+In a labels stage, the map's keys define the label to set and the values are
```

Comment:
```suggestion
In a non_indexed_labels stage, the map's keys define the label to set and the values are
```

Reply:
This is not a `labels` stage we're talking about here, but `non_indexed_labels` stage, right?